cluster/manifests/secretary/deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         args:
         - /meta/credentials
         - --application-id=secretary
-        - --mint-bucket=s3://{{.ConfigItems.MINT_BUCKET}}
+        - --mint-bucket=s3://{{.ConfigItems.gerry_mint_bucket}}
```

Comment:
I would remove the `gerry_` prefix here, it's just a mint bucket. If it's for the case of several different mint buckets then I would prefix it with the application name e.i `secretary`.

What do you think @hjacobs and @linki ?